### DISCUSSION TITLE
Intraday Tick data

### DISCUSSION THREAD

**Discussion by fs4590** [letianzj/QuantResearch#10] [General]
First of all thank you for sharing all these codes .
in your reinforcement_trader you used daily candle sticks , is it possible to use intra day  3000 tick candles ?
I tried using different data set for the NQ futures intra day tick data, but the code only recognized it as spy . is that because the code is designed to use daily data  ?

**Comment by letianzj** [comment]
It surely can. In the code it converts the daily stamp to intraday timestamp via `pd.to_datetime(df.index) + timedelta(hours=15, minutes=59, seconds=59)` where you can use epoch tick stamp instead. 

I didn't find spy hard code except in `load_data()` which you are expected to replace with your own tick data. If you don't mind posting the error message I can take a further look. Thanks.

**Reply by fs4590** [reply]
I will try using epoch tick stamp instead. as far as the code only can see the NQ data as spy data, there was no error message. maybe I forgot to change the symbol. but that is not important now. what matter is your code can handle tick data. I will try that.
 again. thank you so much for help.
